src/test/java/seedu/todo/logic/commands/AddCommandTest.java

```diff
@@ -13,10 +13,67 @@ protected BaseCommand commandUnderTest() {
     }
 
     @Test
-    public void testAdd() throws IllegalValueException {
+    public void testAddTask() throws IllegalValueException {
         setParameter("Hello World");
         execute();
         assertTotalTaskCount(1);
         assertEquals("Hello World", this.getTaskAt(1).getTitle());
```

Comment:
If you're going to run multiple assertions against a single object you can cache it using `ImmutableTask task = this.getTaskAt(1)`. This also makes the code more readable